### DISCUSSION TITLE
자바 버전 변경 후 테스트 실행 오류

### DISCUSSION THREAD

**Discussion by LeeJongbokz** [naver/ngrinder#924] [Q&A]
안녕하세요! 계속 질문을 올려서 죄송합니다 ㅜㅜ
자바 버전을 18->11로 바꾸고 테스트 스크립트를 작성해서  테스트 하는데
다음과 같은 오류가 발생하였습니다.
![image](https://user-images.githubusercontent.com/8718430/196784257-8021fc02-62b7-480d-a6d1-c2329373d1df.png)

혹시 가능하다면 해결책에 대해 알 수 있을까요?
감사합니다!!

**Comment by donggyu04** [comment]
@LeeJongbokz 

처음보는 오류네요.
java8 또는 java11 로 터미널 환경 변경해 주시고(java8 이 더 안정적입니다) 컨트롤러, 에이전트 모두를 다시 구동시켜서 진행해 주시기 바랍니다.
그래도 안된다면 아래 정보 알려주시면 테스트 해보겠습니다.

- os
- 사용 java 버전
- ngrinder 버전

**Reply by LeeJongbokz** [reply]
os: windows
사용 자바 버전: Java 11을 새로 설치했는데, 
                          Runtime Environment는 Java 18로 뜨는 것 같아서,
                          이게 맞을까요?
![image](https://user-images.githubusercontent.com/8718430/197109795-bea05593-5c5a-4be1-bdec-5808f3d54f23.png)
ngrinder 버전: 3.5.6 입니다!

**Reply by junoyoon** [reply]
18 버전이 깔린것 같긴 하군요. 저희가 java 설치 오류까지 대답해 드리는 것은 무리인것 같습니다.
직접 환경을 점검하시기 바랍니다.

**Reply by LeeJongbokz** [reply]
아 넵 알겠습니다!

**Reply by LeeJongbokz** [reply]
아 저 여러 블로그 글 찾아봤는데, 저 버전이 11버전이 맞는 것 같습니다!
https://sparkdia.tistory.com/64
https://crazykim2.tistory.com/478
자바 11 설치 관련된 블로그 포스팅에는 다 이렇게 설명했더라고요
